app/Nova/Team.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+namespace App\Nova;
+
+use Laravel\Nova\Panel;
+use Laravel\Nova\Fields\ID;
+use Illuminate\Http\Request;
+use Laravel\Nova\Fields\Text;
+use Laravel\Nova\Fields\Boolean;
+use Laravel\Nova\Fields\HasMany;
+use Laravel\Nova\Fields\DateTime;
+use Laravel\Nova\Fields\Textarea;
+
+class Team extends Resource
+{
+    /**
+     * The model the resource corresponds to.
+     *
+     * @var string
+     */
+    public static $model = 'App\Team';
+
+    /**
+     * The single value that should be used to represent the resource when being displayed.
+     *
+     * @var string
+     */
+    public static $title = 'name';
+
+    /**
+     * The columns that should be searched.
+     *
+     * @var array
+     */
+    public static $search = [
+        'name',
+        'description',
+    ];
+
+    /**
+     * Get the fields displayed by the resource.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return array
+     */
+    public function fields(Request $request)
+    {
+        return [
+            new Panel('Basic Information', $this->basicFields()),
+
+            new Panel('Communications', $this->commFields()),
+
+            new Panel('Controls', $this->controlFields()),
+
+            new Panel('Metadata', $this->metaFields()),
+
+            HasMany::make('User', 'members'),
+        ];
+    }
+
+    protected function basicFields()
+    {
+        return [
+            Text::make('Name')
+                ->sortable()
+                ->rules('required', 'max:255'),
+
+            Textarea::make('Description')
```

Comment:
Is hiding the field with the `Show Content` action a default? If so, can we override it for this? Since the length is relatively short, I don't see the harm in auto-expanding the description on the front end since nothing else is hidden.

Reply:
It is the default and there is [very little documentation](https://nova.laravel.com/docs/1.0/resources/fields.html#field-types) for that field type.